frontend/src/pages/modelCatalog/screens/ModelDetailsPage.tsx

```diff
@@ -42,18 +42,20 @@ const ModelDetailsPage: React.FC = conditionalArea(
           objectType={ProjectObjectType.registeredModels}
         />
       }
-      empty={model === null}
+      empty={Boolean(modelCatalogSources.error) || model === null}
```

Comment:
Similar to the other page - ApplicationsPage renders errors before it renders empty. We're passing `loadError={modelCatalogSources.error}`, so if `modelCatalogSources.error` is defined, `emptyStatePage` is not going to be rendered regardless of the value of `empty`. So we can change this back to `model === null`.